typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHSTAGEWarpConfig.ts

```diff
@@ -33,6 +36,8 @@ export const ezEthSafes: Record<(typeof chainsToDeploy)[number], string> = {
   taiko: '0x31FF35F84ADB120DbE089D190F03Ac74731Ae83F',
   sei: '0xa30FF77d30Eb2d785f574344B4D11CAAe1949807',
   swell: '0xf40b75fb85C3bEc70D75A1B45ef08FC48Db61115',
+  unichain: '0xa7eccdb9be08178f896c26b7bbd8c3d4e844d9ba',
```

Comment:
deployer key?

Reply:
yeah, could not get a Uni safe up and felt like it didnt matter that much

Reply:
it looks ready. i'll do it now